EIPS/eip-6969.md

```diff
@@ -0,0 +1,126 @@
+---
+eip: 6969
+title: Contract Secured Revenue on an EVM based L2
+status: Draft
+type: Core
+discussions-to: https://t.me/+w1EKtAx824VlZDJh
```

Comment:
Please create a discussion thread on https://ethereum-magicians.org/

Reply:
https://ethereum-magicians.org/t/eip-6969-generalized-csr-protocol/14178

Reply:
Please rename the discussion thread to the assigned EIP number EIP6968